recipes/moltenvk/all/dependencies/dependencies-1.2.0.yml

```diff
@@ -1,4 +1,4 @@
-glslang: "1.3.231.1"
-spirv-cross: "1.3.231.1"
-spirv-tools: "1.3.231.1"
-vulkan-headers: "1.3.231.1"
+glslang: "1.3.239.0"
```

Comment:
We're usually a bit wary of dependency bumps for this recipe - I'll check with @perseoGI tomorrow, but any insight into why this was needed would be appreciated :)

Reply:
This is getting the patch versions in line with the latest 1.2.2 version, approving as-is, but this would not be strictly necessary next time around